categories/ssl_certificates.yaml

```diff
@@ -26,6 +26,7 @@ Ordering:
 - "Renewing a standard SSL Certificate"
 - "How can I select a different SSL certificate domain validation email?"
 - "Choosing the SSL Certificate Names"
+- "I got an ECC-signed certificate but I want a RSA one"
```

Comment:
This should read
"I got an ECC-signed certificate, but I want an RSA one"

Reply:
or "I got an ECC-signed certificate, but want RSA"

Reply:
Updated!

Thank you @itsalyse !